sos/report/plugins/openshift_ovn.py

```diff
@@ -50,21 +51,32 @@ def setup(self):
             'cluster/status OVN_Northbound',
             'ovn-appctl -t /var/run/ovn/ovnsb_db.ctl ' +
             'cluster/status OVN_Southbound'],
-            container='ovnkube-master')
-        self.add_cmd_output([
-            'ovs-appctl -t /var/run/ovn/ovn-controller.*.ctl ' +
-            'ct-zone-list'],
-            container='ovnkube-node')
-        self.add_cmd_output([
-            'ovs-appctl -t /var/run/ovn/ovn-controller.*.ctl ' +
-            'ct-zone-list'],
-            container='ovnkube-controller')
+            container='ovnkube-master',
+            runtime='crio')
+        # We need to determine the actual file name to send
+        # to the command
+        files = glob.glob("/var/run/ovn/ovn-controller.*.ctl")
+        for file in files:
+            self.add_cmd_output([
+                'ovs-appctl -t ' + file +
+                ' ct-zone-list'],
+                container='ovnkube-node',
+                runtime='crio')
+            self.add_cmd_output([
+                'ovs-appctl -t ' + file +
+                ' ct-zone-list'],
+                container='ovnkube-controller',
+                runtime='crio')
```

Comment:
`a_c_o()` takes a list or a string. With this change now doing single files at a time the list is superfluous.

Also, please convert these to f-strings. A while back we made an effort to convert everything we could to f-strings, but I guess we overlooked this.